docs/users-and-roles-swagger.yaml

```diff
@@ -200,6 +201,38 @@ paths:
         responses: {}
         httpMethod: POST
         type: aws_proxy
+  /terms-and-conditions/current:
+    get:
+      summary: Access current terms & conditions
+      operationId: GetCurrentTermsConditionsHandler
+      security:
+        - ExternalCognitoUserPool:
+            - "https://api.nva.unit.no/scopes/third-party/file-upload"
+            - "https://api.nva.unit.no/scopes/third-party/ticket-upsert"
+            - "https://api.nva.unit.no/scopes/third-party/publication-read"
+            - "https://api.nva.unit.no/scopes/third-party/ticket-read"
+            - "https://api.nva.unit.no/scopes/third-party/reporting-read"
+            - "https://api.nva.unit.no/scopes/third-party/publication-upsert"
```

Comment:
necessary?

Reply:
Not sure to be honest. Does we need authentication for this endpoint?

Reply:
no